### ISSUE TITLE
433mhz module 

### ISSUE THREAD

**Issue by AlistairXX** [Zdravevski/talk-to-esp32#1]
Will other type of receivers and transmitters work?
In my country we don’t have access to WL-102-341 modules.

**Comment by Zdravevski** [comment]
Yes, most of them have a similar working principle, I tested it with the one I showed in the video, but you can try it out with others if you have one, and show us the results.
